src/components/SidebarContents/SidebarContents.tsx

```diff
@@ -38,154 +37,341 @@ const SidebarContents = (props: ISidebarContentProps) => {
+                  href="#"
+                  className="hover:underline underline-offset-4"
+                  aria-describedby="toolsAndForms"
+                >
+                  External Certificate
+                </a>
+              </li>
+              <li className="cursor-pointer">
+                <a
+                  href="#"
+                  className="hover:underline underline-offset-4"
+                  aria-describedby="toolsAndForms"
+                >
+                  Claim Form
+                </a>
+              </li>
+            </ul>
+          </li>
+
+          <li className="border border-primary-400 my-4" />
+
+          <li className="cursor-pointer text-lg text-white">
+            <a
+              href="#"
+              className="hover:underline underline-offset-4"
+              aria-describedby="toolsAndForms"
+            >
+              Importance of Waivers
+            </a>
+          </li>
+          <li className="cursor-pointer mt-4 text-lg text-white">
+            <a
+              href="#"
+              className="hover:underline underline-offset-4"
+              aria-describedby="toolsAndForms"
+            >
+              Concussion Training
+            </a>
+          </li>
+          <li className="cursor-pointer mt-4 text-lg text-white">
+            <a
+              href="#"
+              className="hover:underline underline-offset-4"
+              aria-describedby="toolsAndForms"
+            >
+              Concussion Management
+            </a>
+          </li>
+          <li className="cursor-pointer mt-4 text-lg text-white">
+            <a
+              href="#"
+              className="hover:underline underline-offset-4"
+              aria-describedby="toolsAndForms"
+            >
+              Waiver of Minor Participants
+            </a>
+          </li>
+          <li className="cursor-pointer mt-4 text-lg text-white">
+            <a
+              href="#"
+              className="hover:underline underline-offset-4"
+              aria-describedby="toolsAndForms"
+            >
+              Waiver of Adult Participants
+            </a>
+          </li>
+
+          <li className="border border-primary-400 my-4" />
+
+          <li className="cursor-pointer text-lg text-white">
+            <a
+              href="#"
+              className="hover:underline underline-offset-4"
+              aria-describedby="toolsAndForms"
+            >
+              Parade Application Form
+            </a>
+          </li>
+          <li className="cursor-pointer mt-4 text-lg text-white">
+            <a
+              href="#"
+              className="hover:underline underline-offset-4"
+              aria-describedby="toolsAndForms"
+            >
+              Large Event Application Form
+            </a>
+          </li>
+        </div>
+      </ul> */}
+
+      <div className="p-8 w-[360px] lg:w-[715px] min-h-full text-base-content bg-primary text-left">
```

Comment:
`p-8` applied to whole Sidebar content. However, in mobile view, padding should change. Please check Figma for all viewport.